test/dsl_test.go

```diff
@@ -364,6 +398,19 @@ func notExists(filename string) fileOp {
 	}, Defaults}
 }
 
+func mkfileexcl(name string) fileOp {
+	return fileOp{func(c *ctx) error {
+		dir, file := filepath.Split(name)
+		d, _, err := c.getNode(dir, noCreate, resolveAllSyms)
```

Comment:
It might be better to make a `createFileExcl createType` flag to use here, so you can still let the `getNode` function do the filepath stuff.  Any reason not to do it that way?